### DISCUSSION TITLE
Filter in global search..

### DISCUSSION THREAD

**Discussion by Ayushchaudhary-Github** [lnreader/lnreader#505] [Ideas]
1. Maybe a filter where you filter novels with amount of pages they have .

2. By popularity in global search

Need filter to find good novels easyly its hard to find a good novel from a good writter..

**Comment by rajarsheechatterjee** [comment]
> Maybe a filter where you filter novels with amount of pages they have .

This is not possible as the app doesn't know the total chapters in a novel before opening the details screen. 

> By popularity in global search

The default sort should be by popularity only in global search.
